core/src/main/java/com/linecorp/armeria/common/metric/AbstractMetricCollectingBuilder.java

```diff
@@ -36,6 +38,9 @@ public abstract class AbstractMetricCollectingBuilder {
     @Nullable
     private BiPredicate<? super RequestContext, ? super RequestLog> successFunction;
 
+    @Nullable
+    private DistributionStatisticConfig distributionStatisticConfig = MoreMeters.distributionStatisticConfig();
```

Comment:
This value must be used to create `DistributionSummary` or `Timer`. I couldn't see the usage of it. Would you update MetricColllecting{Service,Client,RpcClient} to take the value to record metrics?